hack/components/bump_macvtap.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+MACVTAP_URL=$(cat components.yaml | shyaml get-value components.macvtap-cni.url)
+MACVTAP_COMMIT=$(cat components.yaml | shyaml get-value components.macvtap-cni.commit)
+MACVTAP_REPO=$(echo ${MACVTAP_URL} | sed 's#https://\(.*\)#\1#')
+MACVTAP_PATH=${GOPATH}/src/${MACVTAP_REPO}
+
+echo 'Fetch macvtap-cni sources'
+fetch_component ${MACVTAP_PATH} ${MACVTAP_URL} ${MACVTAP_COMMIT}
+
+rm -rf data/macvtap/*
+echo 'Copy the templates from the macvtap-cni repo ...'
+cp ${MACVTAP_PATH}/templates/namespace.yaml.in data/macvtap/000-ns.yaml
+echo "{{ if .EnableSCC }}" >> data/macvtap/001-rbac.yaml
```

Comment:
maybe make those couple lines (appending 001-rbac.yaml) with HEREDOC and variables ?
(in case you can append with HERDOC)

Reply:
I'm all forward to making the scripts better, but it's not in the scope of this PR, which only intends to split them to separate scripts.